cuda_bindings/cuda/bindings/_path_finder/load_dl_common.py

```diff
@@ -38,3 +35,16 @@ def load_dependencies(libname: str, load_func: Callable[[str], LoadedDL]) -> Non
     """
     for dep in DIRECT_DEPENDENCIES.get(libname, ()):
         load_func(dep)
+
+
+def load_in_subprocess(python_code, timeout=30):
+    # This is to avoid loading libraries into the parent process.
+    return run_python_code_safely(python_code, timeout=timeout)
+
+
+def build_subprocess_failed_for_libname_message(libname, result):
+    return (
+        f"Subprocess failed for {libname=!r} with exit code {result.returncode}\n"
+        f"--- stdout-from-subprocess ---\n{result.stdout}<end-of-stdout-from-subprocess>\n"
+        f"--- stderr-from-subprocess ---\n{result.stderr}<end-of-stderr-from-subprocess>\n"
+    )
```

Comment:
These are not needed now
```suggestion
```

Reply:
Done with commit 5fb2d1fec4932b7b86b4111d87ba73ce8a3c1f71:

`load_in_subprocess()` was removed completely.

`build_subprocess_failed_for_libname_message()` was moved back into test_path_finder_load.py